libraries/hermes/src/Rest/Swagger.ts

```diff
@@ -30,12 +30,33 @@ export class SwaggerGenerator {
       },
     };
     this._parser = new SwaggerParser();
-    this._routerMetadata = routerMetadata;
+    this._routerMetadata = cloneDeep(routerMetadata);
     this._stringifiedGlobalSecurityHeaders = JSON.stringify(
       this._routerMetadata.globalSecurityHeaders,
     );
   }
 
+  cleanup() {
+    this._swaggerDoc = {
```

Comment:
This could be moved into a function to avoid maintaining the same object in two places.